docs/undergrad.yaml

```diff
@@ -180,7 +180,7 @@ Ring Theory:
     $\C$ is algebraically closed: 'complex.exists_root'
     field of fractions of an integral domain: 'fraction_map'
     algebraic elements: 'is_algebraic'
-    transcendental elements:
+    transcendental elements: 'transcendental'
```

Comment:
@PatrickMassot what is supposed to go here according to the French system? Just the definition, or also a proof of existence (either by cardinality or the Liouville proof)...